### ISSUE TITLE
Cannot read properties of null (reading 'name')

### ISSUE THREAD

**Issue by alexfleites** [VemtoOrg/vemto2-issues#22]
**Error Message:**
Cannot read properties of null (reading 'name')

**Stack:**
```
TypeError: Cannot read properties of null (reading 'name')
    at Object.eval (eval at compileAsync (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:302:649), <anonymous>:224:44)
    at lae.compileAsync (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:302:675)
    at lae.compileAsyncWithErrorTreatment (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:302:290)
    at Oy.compile (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:348:14378)
    at Oy.compileWithImports (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:348:13975)
    at async SR.compile (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:350:3029)
    at async SR.compileWithErrorThreatment (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:350:2290)
    at async zi (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:1226:14426)
    at async Be (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:1226:12203)
    at async Y (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.b25c3afb.js:1226:11296)
```

---

Vemto 2.0.4
darwin 23.2.0
Locale: en-US
App version: 2.0.4

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7

**Comment by alexfleites** [comment]
ThxRegards AlexOn Apr 15, 2024, at 4:39 PM, Tiago S. P. Rodrigues ***@***.***> wrote:﻿
Fixed on 2.0.7
